README.md

```diff
@@ -218,6 +218,162 @@ except (ExceptionOne, ExceptionTwo) ...
 
 (espcially when used like `except A, B:` which should be `except (A, B):`.
 
+
+Fix Python 3 failing tests
+==========================
+
+* We try to follow https://docs.python.org/3/howto/pyporting.html
+* some useful info can be found here as well https://portingguide.readthedocs.io/en/latest/index.html
+
+Fixing print statement
+----------------------
+
+```python
+print foo,bar
+```
+=>
+
+```python
+import from __future__ import print_function
+
+
+print (foo,bar)
```

Comment:
no spaces between function name and `(` (actually, only mention the oneliner below)